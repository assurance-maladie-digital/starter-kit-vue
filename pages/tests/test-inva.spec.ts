// @vitest-environment nuxt
import TestInva from '../test-inva.vue';
import { mountSuspended } from '@nuxt/test-utils/runtime';
import { describe, it, expect } from 'vitest';
import { vuetify } from '@/tests/unit/setup';

describe('test inva page should render', () => {
	it('should render', async () => {
		const wrapper = await mountSuspended(TestInva,{
			global: {
				plugins: [
					vuetify,
				],
			},
		});
		expect(wrapper.html()).toMatchSnapshot();

		const h1Text = wrapper.find('h1').text();
		expect(h1Text).toContain('Accueil');
	});
});
