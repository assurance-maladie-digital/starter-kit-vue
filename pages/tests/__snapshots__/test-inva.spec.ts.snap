// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test inva page should render > should render 1`] = `
"<h1 data-v-d90f6bfe="" class="text-primary">Accueil</h1>
<h2 data-v-d90f6bfe="">Test font color scss</h2>
<div data-v-d90f6bfe="" msg="Bonjour">
  <h1 class="text-primary">Hello World!</h1>
</div>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-primary v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Primary </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-secondary v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> secondary </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-info v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Info </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-success v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Success </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-warning v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Warning </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-error v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Error </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-accent v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Accent </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-light v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Light </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-dark v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Dark </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light bg-risquePro v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Risque Pro </span>
  <!---->
  <!---->
</button>
<button data-v-d90f6bfe="" type="button" class="v-btn v-btn--elevated v-theme--light v-btn--density-default v-btn--size-default v-btn--variant-elevated inva"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
  <!----><span class="v-btn__content" data-no-activator=""> Violet inva </span>
  <!---->
  <!---->
</button>
<br data-v-d90f6bfe="">
<br data-v-d90f6bfe="">
<div data-v-1fa5252d="" data-v-d90f6bfe="" class="v-sheet v-theme--light bg-secondary vd-sub-header white--text py-6 px-8" style="width: 100%;">
  <transition-stub data-v-1fa5252d="" name="fade-transition" mode="out-in" appear="false" persisted="false" css="true"><button data-v-1fa5252d="" type="button" class="v-btn v-theme--light v-btn--density-default v-btn--size-small v-btn--variant-text font-weight-regular white--text px-1 vd-sub-header-back-btn mb-1"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
      <!----><span class="v-btn__content" data-no-activator=""><i data-v-1fa5252d="" class="M21,11H6.83L10.41,7.41L9,6L3,12L9,18L10.41,16.58L6.83,13H21V11Z mdi v-icon notranslate v-theme--light v-icon--size-default mr-2" aria-hidden="true"></i> Retour</span>
      <!---->
      <!---->
    </button></transition-stub>
  <div data-v-1fa5252d="" class="vd-sub-header-content d-flex justify-space-between">
    <div data-v-1fa5252d="" class="vd-sub-header-informations d-flex flex-column flex-shrink-0 mr-10">
      <transition-stub data-v-1fa5252d="" name="fade-transition" mode="out-in" appear="false" persisted="false" css="true">
        <h2 data-v-1fa5252d="" class="text-h5 font-weight-bold">Paul Dupont</h2>
      </transition-stub>
      <transition-stub data-v-1fa5252d="" name="fade-transition" mode="out-in" appear="false" persisted="false" css="true">
        <p data-v-1fa5252d="" class="text-h6 font-weight-bold mt-1 mb-0" style="color: rgba(255, 255, 255, .7);">1 69 08 75 125 456 75</p>
      </transition-stub>
    </div>
    <!---->
  </div>
</div>
<br data-v-d90f6bfe="">
<br data-v-d90f6bfe="">
<div data-v-d90f6bfe="" class="v-alert v-theme--light bg-success v-alert--density-default v-alert--variant-flat" role="alert">
  <!----><span class="v-alert__underlay"></span>
  <!---->
  <div class="v-alert__prepend"><i class="mdi-check-circle mdi v-icon notranslate v-theme--light" style="font-size: 28px; height: 28px; width: 28px;" aria-hidden="true" density="default"></i></div>
  <div class="v-alert__content">
    <!---->
    <!----> Message de succ√®s
  </div>
  <!---->
  <!---->
</div>"
`;
